app/javascript/components/ActionItemCards/index.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ActionItemCard.css';
+import {
+  faLeaf,
+  faHome,
+  faPen,
+  faFile,
+  faSmile,
+  faCode,
+} from '@fortawesome/free-solid-svg-icons';
+import { withStyles } from '@material-ui/core/styles';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styles from './styles';
+
+
+class ActionItemCard extends React.Component {
+  constructor(props) {
+    super(props);
+    /*
+    TODO: check props types
+    tempory props rn to test components
+    props:
+    actionItem - Javascripy object
+      title - Assignment title: String
+      description - Short description: String
+      date - Due date of assignment: JavaScript date?
+      category - type of assignment: string??
+      status - status of item: string
+    */
+  }
+
+  // render icon based on category
+  renderIcon() {
+    const AssignmentCategory = this.props.actionItem.category;
+    switch (AssignmentCategory) {
```

Comment:
Since the only thing changing between these is the icon, your switch could just assign a variable to be the appropriate icon, and after the switch just have a single return statement returning the `FontAwesomeIcon` with the appropriate icon